### PR TITLE
Fixed the indentation issue and added sample qos-policy under EP files.

### PR DESCRIPTION
**Pull request by niteshk4-bld**

### CODE REVIEW

File: agent-ovs/lib/EndpointManager.cpp [hunk 1]
```diff
@@ -918,6 +918,23 @@ bool EndpointManager::updateEndpointLocal(const std::string& uuid,
                 l2e->addEpdrEndPointToSecGroupRSrc(sg.toString());
             }
 
+	    const optional<opflex::modb::URI>& qosPol =
```
Comment (tomflynn) [inline agent-ovs/lib/EndpointManager.cpp:921]: The indentation is a bit off for this added block of code

File: agent-ovs/lib/EndpointManager.cpp [hunk 2]
```diff
@@ -918,6 +918,23 @@ bool EndpointManager::updateEndpointLocal(const std::string& uuid,
...
+		    es.endpoint->getQosPol();
+	    if (qosPol)
```
Comment (tomflynn) [inline agent-ovs/lib/EndpointManager.cpp:923]: Please but open braces at the end of line and follow the coding style used elsewhere in the codebase

if () {
    ...
} else {
    ...
}

File: agent-ovs/lib/FSEndpointSource.cpp [hunk 3]
```diff
@@ -208,6 +209,29 @@ void FSEndpointSource::updated(const fs::path& filePath) {
             }
         }
 
+	optional<ptree&> qosPol =
```
Comment (tomflynn) [inline agent-ovs/lib/FSEndpointSource.cpp:212]: Coding style issues. Please update brace placement for 'if' and 'for'

File: agent-ovs/lib/FSEndpointSource.cpp [hunk 4]
```diff
@@ -208,6 +209,29 @@ void FSEndpointSource::updated(const fs::path& filePath) {
...
+		properties.get_child_optional(EP_QOS_POL);
+        if(qosPol)
+        {
+            for (const ptree::value_type &v : qosPol.get())
+            {
+		optional<string> qosPolS =
```
Comment (tomflynn) [inline agent-ovs/lib/FSEndpointSource.cpp:218]: Indent is off here (2 instead of 4 spaces?)

File: agent-ovs/lib/FSEndpointSource.cpp [hunk 5]
```diff
@@ -208,6 +209,29 @@ void FSEndpointSource::updated(const fs::path& filePath) {
...
+                    v.second.get_optional<string>(SEC_GROUP_POLICY_SPACE);
+                optional<string> qosPolName =
+                    v.second.get_optional<string>(SEC_GROUP_NAME);
+                if (qosPolS && qosPolName)
+                {
+                    newep.setQosPol(opflex::modb::URIBuilder()
+				         .addElement("PolicyUniverse")
+					 .addElement("PolicySpace")
+                                         .addElement(qosPolS.get())
```
Comment (tomflynn) [inline agent-ovs/lib/FSEndpointSource.cpp:227]: inconsistent indent

File: agent-ovs/lib/FSEndpointSource.cpp (hunk 3 above)
Comment (niteshk4-bld) [reply agent-ovs/lib/FSEndpointSource.cpp]: done

File: agent-ovs/lib/FSEndpointSource.cpp (hunk 4 above)
Comment (niteshk4-bld) [reply agent-ovs/lib/FSEndpointSource.cpp]: done

File: agent-ovs/lib/FSEndpointSource.cpp (hunk 5 above)
Comment (niteshk4-bld) [reply agent-ovs/lib/FSEndpointSource.cpp:224]: done

File: agent-ovs/lib/EndpointManager.cpp (hunk 2 above)
Comment (niteshk4-bld) [reply agent-ovs/lib/EndpointManager.cpp]: done

**Pull request merged**